src/gpu/nvidia/cudnn_convolution_impl.hpp

```diff
@@ -581,8 +622,32 @@ struct cudnn_convolution_impl_fwd_t : public cudnn_convolution_impl_base_t {
             CUDA_EXECUTE_FUNC(cuMemcpy, (CUdeviceptr)&host_dst_scale,
                     (CUdeviceptr)dst_scale, sizeof(float));
             float inv_scale = 1.0f / host_dst_scale;
-            CUDNN_EXECUTE_FUNC(
-                    cudnnScaleTensor, handle, descs[io::y], y, &inv_scale);
+            if (data_types[io::y] == CUDNN_DATA_INT8) {
+                cudnnOpTensorDescriptor_t opTensorDesc;
+                CUDNN_EXECUTE_FUNC_V(
+                        cudnnCreateOpTensorDescriptor, &opTensorDesc);
```

Comment:
Please don't create objects at execution. oneDNN has a primitive creation where all these objects can be pre-created similar to scratchpad memory, so at execution they are just used without an overhead.

Reply:
This was fixed in the current commit. Thanks for the feedback.